GitOut/Features/Wpf/Commands/Application.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Windows;
+using GitOut.Features.Material.Snackbar;
+
+namespace GitOut.Features.Wpf.Commands
+{
+    public class Application
+    {
+        private readonly ISnackbarService snack;
+
+        public Application(ISnackbarService snack)
+        {
+            this.snack = snack;
+
+            Open = new CompositeCommand<string>(OnOpen);
+            Copy = new CompositeCommand<string>(OnCopy);
+            RevealInExplorer = new CompositeCommand<string>(OnRevealInExplorer);
+        }
+
+        public static CompositeCommand<string>? Open { get; private set; }
```

Comment:
Don't need setters if the property is only set in ctor.

Reply:
it's static, so it does :/
```
1>.\GitOut\Features\Wpf\Commands\Application.cs(17,13,17,17):
 error CS0200: Property or indexer 'Application.Open' cannot be assigned to -- it is read only
```

Reply:
Oh right, didn't notice the static.
I always felt it's not the "right way" to assign a static property from a ctor, but I suppose in this case since Application is more or less a singleton it makes it more OK. Is it possible to add a static ctor to this class instead and assign the properties there?